case-lib/lib.sh

```diff
@@ -90,6 +94,9 @@ func_lib_restore_pulseaudio()
     [[ "${#PULSECMD_LST[@]}" -eq 0 ]] && return
     func_lib_check_sudo
     [[ -f $PULSEAUDIO_CONFIG.bak ]] && sudo mv -f $PULSEAUDIO_CONFIG.bak $PULSEAUDIO_CONFIG
```

Comment:
this line need to remove

Reply:
Yes, but I am afraid that sometimes if we failed to do the rename, the config can still be a backup solution. Let's  leave it as it is.